recipes/metpy/meta.yaml

```diff
@@ -0,0 +1,41 @@
+package:
+  name: metpy
+  version: !!str 0.3.0
```

Comment:
I don't know if this is necessary. @ocefpaf, do you typically just ```"0.3.0"``` it? Or even, just ```0.3.0``` directly.
Either way, one simplification for maintenance of this recipe would be to set a variable up top: https://github.com/conda-forge/staged-recipes/blob/master/ready_recipes/example/meta.yaml#L1